libraries/SPIFFS/src/SPIFFS.h

```diff
@@ -14,26 +14,33 @@
 #ifndef _SPIFFS_H_
 #define _SPIFFS_H_
 
-#include "FS.h"
+#include "vfs_api.h"
```

Comment:
FS should be included. vfs_api is a private header.

Reply:
18 hours ago, you asked me to remove FS.h and FSImpl.h, now you are asking me to add FS.h back in.

If I remove vfs_api.h I will also have to add  FSImpl.h back because SPIFFSImpl is declared in SPIFFS.h.

Reply:
No, I asked about `#include "FSImpl.h"` and `#include "vfs_api.h"`.
FS.h is the only one that should be included :) I guess a bit of misunderstanding.

Reply:
This change requires an additional class, SPIFFFSImpl that inherits VFSImpl, which is declared in vfs_api.h, so something is going to need to include vfs_api.h. Currently this class is declared in SPIFFS.h, which is why SPIFFS.h includes vfs_api.h. How would you have me resolve this?

Reply:
Does this class need to be public? What happens if the declaration goes into the CPP file?

Reply:
It should be fine if I move it to the CPP file.